apps/examples/iotivity_simpleserver/mkfsdata.pl

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/perl
+###########################################################################
+#
+# Copyright 2016 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# tools/mkfsdata.pl
+#
+# Extracted from uIP which has a license that is compatible with NuttX.
```

Comment:
Nuutx license can be removed from this file. I think this file was newly written to copy iotivity cert / dat file data into tinyara file system at run time.